### ISSUE TITLE
Accessibility: Unlabeled close button for up next

### ISSUE THREAD

**Issue by alexstine** [Automattic/pocket-casts-android#1657]
### Description

Currently, the button to close up next does not have a label which is an accessibility issue for TalkBack users.

### Step-by-step reproduction instructions

1. Start TalkBack: https://support.google.com/accessibility/android/answer/6007100?hl=en
2. Open Pocket Casts.
3. Queue some episodes to play next if you do not have any queued already.
4. On the player near the bottom of the screen, just above the tabs, select the Up Next button near the right edge of the screen.
5. The Up Next popup appears.
6. Located just to the right of the Up Next title, notice the button to close does not have an accessibility label.
7. Please label this button with "Close" or "Close Up Next".
https://support.google.com/accessibility/android/answer/7158690?hl=en

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Pixel 7 with Android 14.
Pocket Casts version: 7.53
